pages/dkp/kommander/2.2/cli/dkp/update/nodepool/preprovisioned/index.md

```diff
@@ -26,9 +26,9 @@ dkp update nodepool preprovisioned [flags]
   -h, --help                        help for preprovisioned
       --kubeconfig string           Path to the kubeconfig for the management cluster. If unspecified, default discovery rules apply.
       --kubernetes-version string   Kubernetes version
-  -n, --namespace string            If present, the namespace scope for this CLI request.
+  -n, --namespace string            If present, the namespace scope for this CLI request. (default "default")
```

Comment:
didn't pull in changes in https://github.com/mesosphere/kommander-cli/pull/410 🤔 maybe it'll get pulled in tomorrow? (will double check on tomorrow's automated pr)